src/checkout.sh

```diff
@@ -59,4 +59,7 @@ fi
 
 git checkout "$MERGE_REF"
 
+# Copy common post-deploy scripts
+cp /home/circleci/source/tasks/post-deploy/* /home/circleci/merge/post_deploy_scripts
```

Comment:
This will override the NRO tasks with the base tasks, which is not the behaviour we're after I believe.  I think it's a better option to overwrite base with NRO tasks, allowing the freedom to inherit from, but still override the default tasks if we require.

See https://github.com/greenpeace/planet4-builder/blob/master/src/bake.sh#L20 

After the $GIT_SOURCE and $MERGE_SOURCE repository are checked out, the MERGE repo (NRO) is copied over the top of the SOURCE repo.

We just need to persist these changes from the build job to the deploy job in some way - either by storing it in the workspace, eg at https://github.com/greenpeace/planet4-flibble/blob/develop/.circleci/config.yml#L77 or by reproducing this merge behaviour in the deploy job, each with pros and cons at this point.

Reply:
Yes, but this would be need to be done in all NRO repos.
Koyan stated that there is an easy way to do that though.

We can still copy base-fork scripts and not overwrite existing NRO scripts at this step.
Or we can follow a naming convention to ensure the execution order we want.

I would try pesisting to workspace approach

Reply:
It's the order of the copy that's the first problem: copy NRO over base to have logical inheritance, not the other way around.

Yes there's an automated method for updating all NRO CI configs.

Reply:
Another option is to bake the post-deploy tasks into the container images and execute them directly from there?